### PR TITLE
Added failover method of connecting wifi

### LINKED ISSUE #146: To add a fail over method of connecting wifi
**Actual Behaviour**

Currently no failover method

**Expected Behaviour**

If there are wrong credentials entered by the user, the device should check 3 times and then prompt again

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #146 

Changes
This adds a failover method to check if the wifi is connected to the internet or not

### CODE REVIEW

File: access_point/wifi_test.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+WIFI=''
```
Comment (prateekiiest) [inline access_point/wifi_test.sh]: a variable name like `WIFI_ON` maybe?

File: access_point/wifi_test.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    echo "connection successful"
+    WIFI='1'
+elif ping -q -t 5 -w1 -c1 8.8.8.8 t
+then
+    echo "connection successful"
+    WIFI='1'
+
+else
+    echo "wifi not connected"
+    WIFI='0'
```
Comment (prateekiiest) [inline access_point/wifi_test.sh]: @stealthanthrax how did you plan to check for 3 times password failure you discussed here https://github.com/fossasia/susi_linux/issues/146.

I suppose this only checks the connection establishment, right?

File: access_point/wifi_test.sh (hunk 1 above)
Comment (cweitat) [reply access_point/wifi_test.sh]: +1

File: access_point/wifi_test.sh (hunk 2 above)
Comment (cweitat) [reply access_point/wifi_test.sh]: @prateekiiest i think he meant check 3 times to ensure that there's is a connection else password is wrong

Comment (cweitat) [reply access_point/wifi_test.sh]: but then again the code doesn't does what I just specified. 
@stealthanthrax either update your code or update your issue pls. Thanks

Comment (stealthanthrax) [reply access_point/wifi_test.sh]: @cweitat , i'll be updating the code

**Pull request merged**